can-stache-bindings.js

```diff
@@ -1283,6 +1265,29 @@ var makeDataBinding = function(node, el, bindingData) {
 	// This completes the binding.  We can't call it right away because
 	// the `viewModel` might not have been created yet.
 	var completeBinding = function() {
+
+		//!steal-remove-start
+		if(
+			bindingInfo.child === "viewModel" &&
+			!isBindingsAttribute(bindingInfo.bindingAttributeName)
+		) {
+			// Check to see if the viewModel has the attribute name as a defined property
+			var viewModel = bindingData.getViewModel();
+			if (canReflect.hasOwnKey(viewModel, bindingInfo.bindingAttributeName) === false) {
```

Comment:
could this just be `!canReflect.hasOwnKey(...)`?

Reply:
Using === is faster